### DISCUSSION TITLE
Mux-Player end time option

### DISCUSSION THREAD

**Discussion by ewettach** [muxinc/elements#569] [Q&A] [answered]
Hi,  we are currently not using mux-player but interested in moving to it.  We are currently using video-js player.  The only thing holding us back from moving to mux-player is the ability to "trim" a clip.  From what I can tell we can use **start-time** to set the point we want the video playback to start, however, I don't see an **end-time** where we can specify an endpoint.  

What we are trying to do is let's say the recording is an hour long and has 10 minutes at the beginning of the Mux asset and 10 minutes at the end of the Mux asset that is extra pre/post roll that we don't need and want to trim it.  Instead of creating a new asset, we are just using video-js start-time and end-time to tell the player when to start and end.  

Is this possibe now with mux-player?  Any ideas would be appreciated.   Thanks!

**Comment by luwes** [comment, marked as answer]
Hi @ewettach! Thanks for the question, this is not possible out of the box but we recently created a small script for exactly this use case. Great timing! 

Have a look at https://github.com/muxinc/media-offset for the repo.
Here for some examples https://media-offset-mux.vercel.app/examples/mux-player.html

---

One small gotcha is that the time slider preview thumbnails might be offset incorrectly as the offset would not be incorporated in this logic. We will attempt to fix this soon in https://github.com/muxinc/media-chrome
